torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -629,12 +627,12 @@ void ScriptModuleSerializer::writeByteCode(
     const bool save_mobile_debug_info) {
   std::vector<c10::IValue> elements;
   BackendDebugInfoRecorder debug_info_recorder;
-  elements.emplace_back(
-      static_cast<int64_t>(caffe2::serialize::kProducedBytecodeVersion));
+  int64_t version_to_write = caffe2::serialize::kProducedBytecodeVersion;
```

Comment:
```suggestion
  cosnt int64_t version_to_write = caffe2::serialize::kProducedBytecodeVersion;
```

or just use kProducedBytecodeVersion directly

Reply:
Yeah I should use kProducedBytecodeVersion directly. It's remainders from last commit. Thanks!